test/android/template/AndroidManifest.xml

```diff
@@ -11,7 +11,7 @@
                  android:theme="@android:style/Theme.NoTitleBar.Fullscreen"
                  android:hardwareAccelerated="true"
                  $(ANDROID_DEBUGGABLE) >
-        <activity android:name=".MainActivity"
+        <activity android:name=".$(ACTIVITY_CLASS_NAME)"
```

Comment:
This will not work when user customizes package id and wants to use NimxActivity. The dot should be inside the ACTIVITY_CLASS_NAME var.